modules/log6x-rn-6.0.2.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+// log6x-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-0-2_{context}"]
+= Logging 6.0.2
+This release includes link:https://access.redhat.com/errata/RHBA-2024:10051[RHBA-2024:10051].
+
+[id="logging-release-notes-6-0-2-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, Loki did not correctly load some configurations, which caused issues when using Alibaba Cloud or IBM Cloud object storage. This update fixes the configuration-loading code in Loki, resolving the issue. (link:https://issues.redhat.com/browse/LOG-5325[LOG-5325])
+
+* Before this update, the collector would discard audit log messages that exceeded the configured threshold. This modifies the audit configuration thresholds for the maximum line size as well as the number of bytes read during a read cycle. (link:https://issues.redhat.com/browse/LOG-5998[LOG-5998])
+
+* Before this update, the Cluster Logging Operator did not watch and reconcile resources associated with an instance of a ClusterLogForwarder like it did in prior releases. This update modifies the operator to watch and reconcile all resources it owns and creates. (link:https://issues.redhat.com/browse/LOG-6264[LOG-6264])
+
+* Before this update, log events with an unknown severity level sent to Google Cloud Logging would trigger a warning in the vector collector, which would then default the severity to 'DEFAULT'. With this update, log severity levels are now standardized to match Google Cloud Logging specifications, and audit logs are assigned a severity of 'INFO'. (link:https://issues.redhat.com/browse/LOG-6296[LOG-6296])
```

Comment:
Since `'` isn't a markup we use:
```suggestion
* Before this update, log events with an unknown severity level sent to Google Cloud Logging would trigger a warning in the vector collector, which would then default the severity to `DEFAULT`. With this update, log severity levels are now standardized to match Google Cloud Logging specifications, and audit logs are assigned a severity of `INFO`. (link:https://issues.redhat.com/browse/LOG-6296[LOG-6296])
```

Reply:
Work to fix this is covered in [OBSDOCS-1534](https://issues.redhat.com/browse/OBSDOCS-1534).